sparse/tests/test_coo.py

```diff
@@ -314,11 +314,11 @@ def test_dot_nocoercion():
         assert_eq(operator.matmul(a, lb), operator.matmul(sa, lb))
 
 
-@pytest.mark.parametrize('a_ndim', [1, 2])
-@pytest.mark.parametrize('b_ndim', [1, 2])
+@pytest.mark.parametrize('a_ndim', [1, 2, 3])
+@pytest.mark.parametrize('b_ndim', [1, 2, 3])
 def test_kron(a_ndim, b_ndim):
-    a_shape = (10, 11)[:a_ndim]
-    b_shape = (12, 13)[:b_ndim]
+    a_shape = (3, 4, 5)[:a_ndim]
+    b_shape = (4, 5, 3)[:b_ndim]
```

Comment:
It'd be maybe good to test:
- Using dimensions that don't ever match up (`len(set(a_shape).intersection(b_shape)) == 0`) to ensure no accidental reliance on broadcasting aligning. Perhaps `(3, 4, 5)`, and `(6, 7, 8)`.
- Cases with a nonzero fill value

Reply:
The first one might be being overly paranoid though, feel free to ignore.

Reply:
Ah, yes. My algorithm only works with a zero fill-value.